### ISSUE TITLE
SMB Path Support

### ISSUE THREAD

**Issue by ultraviolet7** [TeamAmaze/AmazeFileManager#286]
Adding a new SMB connection using the "use custom IP" option does not appear to allow specifying the server and path in the "Server IP Address" field.  When a '/' is used in the field the connection fails.  When editing an SMB connection that contains a '/', the '/' and any text after is dropped.  If "%2F" is used instead of '/' the value is retained during edits to the connection but the connection still fails.

Specifying the server and path is necessary for servers that do not advertise their shares.  Adding an SMB connection with a server and path is a feature that used to be supported by AmazeFileManager.

**Comment by pradyukrish** [comment]
Thanks for reporting. I'll look into this. Btw, which version of Amaze are you using?

**Comment by ultraviolet7** [comment]
I am using version 3.1.0 from the Google Play Store.  I tried a few more things and if I use "//" instead of "/" it works correctly.  I guess the single slash is not being escaped somewhere.

**Comment by pradyukrish** [comment]
'/' or '\' ? Backslash has to be escaped. Not forward slash.

**Comment by apemanzilla** [comment]
Could this be re-opened? I can't get it to connect to my hidden share with any combination of slashes in the server field, using v3.2.1.

**Comment by Phazeshift** [comment]
I was able to workaround this and access a hidden share in 3.2.1 using ultraviolet7s suggestion of entering <ipaddress>//<hiddensharename> in the server ip address field, unfortunately it doesn't reload correctly if you edit the smb connection.

**Comment by galearned** [comment]
Good evening:  I am trying to use version 3.5.0 (Custom IP) to connect to a windows server with a classic shared folder.  I have tried the usual "IPaddress\shareName" with username and password and, the app does not proceed.  I have used "IPaddress/shareName" and double slashes as well and, all the app does is spin.  I am able to use two other file manager apps which work fine and, my Workstation so the server works.  There is no Domain and, if I go back to the menu and edit the SMB screen, the share and slash has been shortened back to just the IP address.
Regards
